repos/spack_repo/builtin/packages/amdlibm/package.py

```diff
@@ -33,10 +34,11 @@ class Amdlibm(SConsPackage):
     license("BSD-3-Clause")
 
     version(
-        "5.0",
-        sha256="ba1d50c068938c9a927e37e5630f683b6149d7d5a95efffeb76e7c9a8bcb2b5e",
+        "5.1",
+        sha256="7acf2c98469353b60a59fad167a98e1ae689055a3faf8352a254832145c9d59e",
         preferred=True,
     )
```

Comment:
(You guessed it) Preferred doesn't do anything when it's on the most recent version.